packages/atomic/cypress/e2e/smart-snippet.cypress.ts

```diff
@@ -30,458 +30,386 @@ function buildAnswerWithHeight(height: number) {
+            'snippet-collapsed-height': 150,
+          },
+        })
+      )
+      .init();
+
+    SmartSnippetAssertions.assertShowMore(false);
+    SmartSnippetAssertions.assertShowLess(false);
   });
 
-  describe('when the snippet height is greater than maximumHeight', () => {
+  it('when the snippet height is greater than maximumHeight', () => {
     const height = 300;
     const heightWhenCollapsed = 150;
-    beforeEach(() => {
-      new TestFixture()
-        .with(
-          addSmartSnippet({
-            snippet: {
-              ...defaultSnippet,
-              answer: buildAnswerWithHeight(height),
-            },
-            props: {
-              'maximum-height': height - 1,
-              'collapsed-height': heightWhenCollapsed,
-            },
-          })
-        )
-        .init();
-    });
-
-    it('should render correctly', () => {
-      SmartSnippetAssertions.assertShowMore(true);
-      SmartSnippetAssertions.assertShowLess(false);
-      SmartSnippetAssertions.assertAnswerHeight(heightWhenCollapsed);
-    });
 
+    new TestFixture()
+      .with(
+        addSmartSnippet({
+          snippet: {
+            ...defaultSnippet,
+            answer: buildAnswerWithHeight(height),
+          },
+          props: {
+            'maximum-height': height - 1,
+            'collapsed-height': heightWhenCollapsed,
+          },
+        })
+      )
+      .init();
+
+    // before expand
+    SmartSnippetAssertions.assertShowMore(true);
+    SmartSnippetAssertions.assertShowLess(false);
+    SmartSnippetAssertions.assertAnswerHeight(heightWhenCollapsed);
     CommonAssertions.assertAccessibility(smartSnippetComponent);
-
-    it('should work correctly when pressing show more and show less', () => {
-      SmartSnippetSelectors.showMoreButton().click();
-      SmartSnippetSelectors.body().should('have.attr', 'expanded');
-      SmartSnippetAssertions.assertShowMore(false);
-      SmartSnippetAssertions.assertShowLess(true);
-      SmartSnippetAssertions.assertAnswerHeight(height);
-      SmartSnippetSelectors.showLessButton().click();
-      SmartSnippetSelectors.body().should('not.have.attr', 'expanded');
-      SmartSnippetAssertions.assertShowMore(true);
-      SmartSnippetAssertions.assertShowLess(false);
-      SmartSnippetAssertions.assertAnswerHeight(heightWhenCollapsed);
-    });
+    SmartSnippetSelectors.showMoreButton().click();
+
+    // after expand
+    SmartSnippetSelectors.body().should('have.attr', 'expanded');
+    SmartSnippetAssertions.assertShowMore(false);
+    SmartSnippetAssertions.assertShowLess(true);
+    SmartSnippetAssertions.assertAnswerHeight(height);
+    SmartSnippetSelectors.showLessButton().click();
+
+    // after collapse
+    SmartSnippetSelectors.body().should('not.have.attr', 'expanded');
+    SmartSnippetAssertions.assertShowMore(true);
+    SmartSnippetAssertions.assertShowLess(false);
+    SmartSnippetAssertions.assertAnswerHeight(heightWhenCollapsed);
   });
```

Comment:
I'm not sure about this part: I think there's value to test interaction 'separately'

Reply:
It execute much faster ! 🏃‍♂️